recipes/vegafusion-python-embed/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "vegafusion-python-embed" %}
+{% set name_under = "vegafusion_python_embed" %}
+{% set version = "0.0.4_rc.1" %}
```

Comment:
is this an RC?

Reply:
Thanks for the review @wolfv!  I agree this should build from an actual release if at all possible.

Reply:
Yeah, this is an RC because the current stable version doesn't have sdist bundles published.  If this approach looks good otherwise, I can publish a new version.

Reply:
I think if you're ready to release `0.0.4` proper then this recipe is in good shape to be merged.

Ofc, it's always possible further :eyes: will pick up some more issues but it's easier for others to review when it's ready to merge (otherwise it would need a further review when it was ready)

Reply:
Tests running on version 0.0.4